extensions/wikia/PremiumPageHeader/styles/_wiki-header.scss

```diff
@@ -74,7 +83,20 @@
 	-webkit-font-smoothing: antialiased;
 	-moz-osx-font-smoothing: grayscale;
 
+	&::before {
+		background: linear-gradient(to right, rgba($color-buttons, .5), rgba($color-buttons, .5)),
+					linear-gradient(to left, rgba($color-buttons, .0) 200px, rgba($color-buttons, 1) $image-width);
+		content: '';
+		display: block;
+		height: 100%;
+		position: absolute;
+		width: 100%;
+		z-index: -1;
+	}
+
 	.wordmark {
+		// !important to avoid showing bg image from parent when wordmark has opacity
+		background: $color-buttons !important;
```

Comment:
no, it does not affect the gradient, at least I don't see the difference in gradient. And taking into account that we need to use !important here, I'd prefer to set this colour explicitly to make sure that it is the same as colour of the rest of header